scripts/dex-router/src/index.ts

```diff
@@ -321,18 +322,24 @@ export class DexRouter {
     instructions: TransactionInstruction[],
     routerLookupTableAddresses: PublicKey[]
   ): Promise<VersionedTransaction> {
-    if (!this.recentBlockhash[this.chainId]) {
+    if (!this.latestChainUpdate[this.chainId]) {
       console.log(
         `No recent blockhash for chain ${this.chainId}, getting manually`
       );
-      this.recentBlockhash[this.chainId] = (
+      this.latestChainUpdate[this.chainId] = (
         await this.connectionSvm.getLatestBlockhash("confirmed")
       ).blockhash;
+    } else {
+      const feeInstruction = ComputeBudgetProgram.setComputeUnitPrice({
```

Comment:
we are handling the compute budget instruction somewhere else in the code via what jupiter gives us and with the filterComputeBudget call, pls remove the use of the jupiter compute budget ix and modify the filter ix

Reply:
i think it's fine to keep the filter ix as a way to cap against server sending outrageous priority fees. you can also add this fee instruction construction to the file `computeBudget.ts` since that already exists and would be a better place for it to live

Reply:
another thought: you could get the jupiter compute budget ixs and filter out the set price ix for this one. that way, you could still have a set compute unit limit ix. not necessary, feel free to ignore this